src/scheduling/dimacs_change.h

```diff
@@ -6,6 +6,8 @@
 
```

Comment:
It's the superclass for the concrete types of change, though, so I think this is okay?